openedx/features/course_experience/static/course_experience/fixtures/welcome-message-fragment.html

```diff
@@ -1,6 +1,9 @@
 <div class="welcome-message">
     <div class="dismiss-message">
-        <button type="button" class="btn-link">${_("Dismiss")}</button>
+        <button type="button" class="btn-link">
+            <span class="sr">Dismiss</span>
+            <span class="icon fa fa-lg fa-times" aria-hidden="true"></span>
```

Comment:
moved to in-context comment - 
NIT / I would also suggest we match the font-size of the course tools icons of 16px by removing fa-lg to this and the other template